docs/SECURITY.md

```
@@ -18,15 +19,16 @@ responsible disclosure and will make every effort to acknowledge your
 contributions.
 Unfortunately, SORMAS does not offer a paid bug bounty programme or other forms of compensation.
 
-Report security bugs by emailing at **security@sormas.org**.
+Report security bugs by emailing at **info@sormas.org**.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)